pkgs/by-name/kh/khal/package.nix

```
@@ -31,20 +31,16 @@ let
 in
 python.pkgs.buildPythonApplication rec {
   pname = "khal";
-  version = "0.11.3";
+  version = "0.11.4";
```

```suggestion
  version = "0.13.0";
```
setuptools is at version 78.1.0 on master now.

---

Thanks for the notice!